compiler/src/utils/config.re

```diff
@@ -268,6 +244,56 @@ let with_cli_options = (term: 'a): Cmdliner.Term.t('a) => {
   folded;
 };
 
+let with_unapplied_cli_options = (term: 'a): Cmdliner.Term.t('a) => {
```

Comment:
I don't love having to need this, but it was the only way I could get `with_used_args` to be populated.